main.go

```diff
@@ -149,17 +152,18 @@ func New() *Application {
 	credsCommand.Action(credsCommand.Download).Hidden()
 
 	envCommand := cap.NewEnvCommand(ctx, "env", "show source command for setting credential environment."+
-		" Use with: eval `carina env <cluster-name>`")
+		" Use with: eval \"$( carina env <cluster-name> )\"")
```

Comment:
This will need updates in tandem with everything else for https://github.com/getcarina/carina/issues/57.